client/src/modules/multiple_payroll/multiple_payroll.js

```diff
@@ -186,6 +185,7 @@ function MultiplePayrollController(
   //
   vm.putOnWaiting = function putOnWaiting() {
     const employees = vm.gridApi.selection.getSelectedRows();
+    vm.getSelectedEmployes = employees;
```

Comment:
A minor English typo here - this should be called `getSelectedEmployees`.  You are missing an `e` at the end.